src/UI/PrimaryButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from "react"
+import Button from "@material-ui/core/Button"
+import { makeStyles } from "@material-ui/core/styles"
+import Icon from "@material-ui/core/Icon"
+import DoubleArrowIcon from "@material-ui/icons/DoubleArrow"
+import MeetingRoomIcon from "@material-ui/icons/MeetingRoom"
+import { RegexContext } from "../store/regex-context"
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    margin: theme.spacing(1),
+  },
+  custom: {
```

Comment:
looks like this custom property is static, so it would be better to replace custom --> bigBtn

custom would be great choice if you'd send values through props